apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -1,7 +1,11 @@
 import * as React from 'react';
 import { Notification } from '@fluentui-react-native/notification';
 import { Test, TestSection, PlatformStatus } from '../Test';
-import { Animated, Button, Easing, View } from 'react-native';
+import { Animated, Button, Easing, Switch, TextInput, View } from 'react-native';
+import { Text } from '@fluentui-react-native/experimental-text';
+import { StyledPicker } from '../Common/StyledPicker';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+import { NotificationVariant, NotificationVariants } from '../../../../../../packages/components/Notification/src/Notification.types';
```

Comment:
Fix import here

Reply:
Using ~/ doesn't seem to work. Any other ideas?